.github/workflows/release.yml

```diff
@@ -72,9 +74,80 @@ jobs:
           name: Windows Release binaries
           path: ${{github.workspace}}/*.zip
 
+  linux-build:
+    name: "Build on Ubuntu"
+    runs-on:
+      - ubuntu-latest
+
+    permissions:
+      actions: write
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: CMake
+        run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=Release -DNOTEST=1
+
+      - name: Build
+        run: |
+          cmake --build ${{github.workspace}}/build --config Release
+          cpack --config ${{github.workspace}}/build/CPackConfig.cmake -C Release -G ZIP
+
+      - name: Test  # don't release unless tests pass
+        working-directory: ${{github.workspace}}/build
+        run: ctest -C Release -L anyplatform --output-on-failure
+
```

Comment:
```suggestion
```

I see this is identical to the eGrabber release file, which highlights a mistake I've made. We've configured the build to skip tests, yet we're running ctest here🤦. We don't have any tests to run for this platform anyway.

Reply:
Ah, good catch. In practice, it's possible some tests run (the test-acquire-core-libs-* ones), but they're not of interest, so we should still remove this anyway.